components/members/main/MemberDetail/index.tsx

```diff
@@ -147,12 +158,13 @@ const MemberDetail: FC<MemberDetailProps> = ({ memberId }) => {
               )}
             </>
           )}
-
-          <InfoContainer style={{ gap: '30px' }}>
-            <InfoItem label='생년월일' content={convertBirthdayFormat(profile?.birthday)} />
-            <InfoItem label='사는 지역' content={profile?.address ?? ''} />
-            <InfoItem label='학교 / 전공' content={`${profile?.university ?? ''} ${profile?.major ?? ''}`} />
-          </InfoContainer>
+          {profile?.birthday && profile?.address && profile?.university && (
```

Comment:
p1; 이러면 생년월일은 있고, 사는 지역은 없으면 생년월일도 같이 안나오지 않나요!? 조건을 이렇게 변경해야할 것 같아요.

```suggestion
          {profile?.birthday || profile?.address || profile?.university || (
```

Reply:
바보같은 실수네여 . . 감사합니다